packages/wrangler/src/secret/index.ts

```diff
@@ -282,11 +282,17 @@ export const secret = (secretYargs: CommonYargsArgv) => {
 			"list",
 			"List all secrets for a Worker",
 			(yargs) => {
-				return yargs.option("name", {
-					describe: "Name of the Worker",
-					type: "string",
-					requiresArg: true,
-				});
+				return yargs
+					.option("name", {
+						describe: "Name of the Worker",
+						type: "string",
+						requiresArg: true,
+					})
+					.option("pretty", {
```

Comment:
instead of adding this as a cmd option, why not pretty print by default? Correct me if I'm wrong, but I'm assuming we're adding this so that users can make sense of whatever `JSON.stringify(await fetchResult(url), null, "  ")` will print? If we log things that are not easy to decipher, IMHO we either should not log those at all, or log them in a format that is helpful for our users. But maybe I'm missing context and am way off 😅 ?

Reply:
My thinking is that this would be a breaking change doing it by default, we could have users expecting it to be a JSON today and parsing it. Imo, not an invalid use case, means you could avoid pushing secrets that already exist

Reply:
what prompted us to add `--pretty`? what's the use case?

Reply:
Creates a better view for people, dumping JSON out isn't great. Moving the default view in the next major version to the pretty one is the goal. Just wasn't done to avoid breaking changes here.

Reply:
right, thx for the context. Totally up to you, but maybe for consistency purposes, would it make sense to use a similar option as we do for [tailing](url)? That's really the only place, AFAICT, where we're giving a choice between `json` and `pretty` output. So smth like 👇  as per https://developers.cloudflare.com/workers/wrangler/commands/#tail
<img width="283" alt="Screenshot 2024-06-20 at 13 33 56" src="https://github.com/cloudflare/workers-sdk/assets/4638332/84379b47-2645-419d-a2f7-845906ee119e">

Reply:
yep, agreed, done